bdns/dns.go

```diff
@@ -432,11 +432,11 @@ func (dnsClient *DNSClientImpl) LookupHost(ctx context.Context, hostname string)
 
 // LookupCAA sends a DNS query to find all CAA records associated with
 // the provided hostname.
-func (dnsClient *DNSClientImpl) LookupCAA(ctx context.Context, hostname string) ([]*dns.CAA, error) {
+func (dnsClient *DNSClientImpl) LookupCAA(ctx context.Context, hostname string) ([]*dns.CAA, []*dns.CNAME, error) {
```

Comment:
Can you describe the `[]*dns.CNAME` return in the comment for `LookupCAA`?

Reply:
Done.